eth/api_backend.go

```diff
@@ -295,9 +302,10 @@ func (b *EthAPIBackend) SendTx(ctx context.Context, signedTx *types.Transaction)
 				}
 			}
 		}
-
+		return b.eth.syncService.ApplyTransaction(signedTx)
 	}
-	return b.eth.syncService.ApplyTransaction(signedTx)
+	// OVM Disabled
```

Comment:
Not really sure what this means. Maybe due to unfamiliarity, but maybe the comment can be a little more descriptive.

Reply:
The comment ovm disabled is used throughout the codebase to indicate when particular changes apply when `USING_OVM` is unset. This will result in geth like behavior. Setting `USING_OVM` will cause ovm behavior